### DISCUSSION TITLE
【Statistic】是否可以支持简化配置？

### DISCUSSION THREAD

**Discussion by dinfer** [antvis/G2Plot#2541] [General]
例如[进度环图](https://g2plot.antv.vision/zh/examples/tiny/ring-progress#ring-progress-style)中`statistic.title`是不是可以直接设置文本。
有些情况是需要formatter函数执行，但是简单情况下，没必要用函数设置常量字符串，能不能加个静态内容的字段？

**Comment by visiky** [comment]
#2567 is satisfied
